ui/lib/components/CardTableV2/index.tsx

```diff
@@ -70,28 +70,22 @@ function renderColumnVisibilitySelection(
       visibleColumnKeys![c.key] = true
     })
   }
-  return (
-    <>
-      {columns.map((column) => (
-        <div key={column.key}>
-          <Checkbox
-            checked={visibleColumnKeys![column.key]}
-            onChange={(e) => {
-              if (!onChange) {
-                return
-              }
-              onChange({
-                ...visibleColumnKeys!,
-                [column.key]: e.target.checked,
-              })
-            }}
-          >
-            {column.name}
-          </Checkbox>
-        </div>
-      ))}
-    </>
-  )
+  return columns.map((column) => (
+    <Checkbox
+      checked={visibleColumnKeys![column.key]}
+      onChange={(e) => {
+        if (!onChange) {
+          return
+        }
+        onChange({
+          ...visibleColumnKeys!,
+          [column.key]: e.target.checked,
+        })
+      }}
+    >
+      {column.name}
+    </Checkbox>
+  ))
```

Comment:
The original implementation can't apply the correct style inside the `<Menu>`, even change the `<div key={column.key}>` to `<Menu.Item .../>`. the padding or margin will be lost, not sure why.